MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -528,9 +881,18 @@ int main(int argc, char * argv[])
                 HRESULT hrMountCIM = MsixCoreLib::MountCIM(cli.GetMountImagePath(), volumeId);
                 if (FAILED(hrMountCIM))
                 {
+                    errorCode = msixmgr::GetErrorCodeFromHRESULT(hrMountCIM);
+                    errorDesc = L"Failed Mounting the CIM file. HRESULT " + errorCode + L".";

```

Comment:
Same as above

Reply:
As discussed, all the Telemetry variables will have the same Error Message as the actual Console statements. The change for outputting the Error Message on Console from a variable will be done as part of Error Message Improvement Task that will be taken later.